refactor: Улучшение логики отправки сообщений и детекции

GachiSlave opened the pull request:
refactor: Улучшение логики отправки сообщений и детекции освободившихся мест

- Исправлена логика отправки скриншотов с рамкой для свободных парковочных мест.
- Обновлена логика для обработки случаев с несколькими свободными местами.
- Вынесено за основной цикл: Закрытие всех окон с кадрами после завершения упрощает управление ресурсами.
- Добавлен линтер для поддержания качества кода и набросок документации README.
- Использован black

GachiSlave committed:
refactor: Улучшение логики отправки сообщений и детекции
освободившихся мест

- Исправлена логика отправки скриншотов с рамкой для свободных
  парковочных мест.
  - Обновлена логика для обработки случаев с несколькими свободными
    местами.
    - Вынесено за основной цикл: Закрытие всех окон с кадрами после
      завершения упрощает управление ресурсами.
      - Добавлен линтер для поддержания качества кода и набросок
	документации README.

DanilaAkh committed:
feat: Исправление стиля перед разделением  кода на отдельные модули

-	Исправлена стилистика когда (лишние/недостающие переменные и т.д.)
-	Добавлены комментарии к модулям
-	Упрощены некоторые функции в соответствии с новыми переменными

DanilaAkh reviewed (approved):
-	Исправлена стилистика кода (лишние/недостающие переменные и т.д.)
-	Добавлены комментарии к модулям
-	Упрощены некоторые функции в соответствии с новыми переменными

AlChudi committed:
feat: Обновление документации

- Обновлен пункт установки зависимостей и запуска виртуальной среды
- Запуск и установка подготовлены для Linux и Windows
- Добавлены более подробные комментарии по настройке конфигурации телеграмм-бота

AlChudi reviewed (approved):
Обновление документации
